storybook.test.ts

```diff
@@ -1,4 +1,13 @@
 import initStoryshots from "@storybook/addon-storyshots";
 import "jest-styled-components";
+import { axeTest } from "@storybook/addon-storyshots-puppeteer";
+import path from "path";
+
+initStoryshots({
+  suite: "A11y checks",
+  test: axeTest({
+    storybookUrl: `file://${path.resolve(__dirname, "./storybook-static")}`
+  })
+});
```

Comment:
This makes the tests take over two minutes on my computer vs about 20 seconds without. We should have this run in a separate test script than the unit tests since unit tests are supposed to run fairly quickly.

Reply:
Split up in 641de5f11cd6e8eb781d1433a13c2f3628191471